lug/config.bootstrap.yaml

```diff
@@ -1,166 +1,160 @@
-interval: 10 # Interval between pollings
-loglevel: 4 # 1-5
 concurrent_limit: 1
-# Prometheus metrics are exposed at http://exporter_address/metrics
+dummy:
+  oneshot_common:
+    retry: 5
+    retry_interval: 60
 exporter_address: :8081
-
-# Address where JSON API will be served
+interval: 10
 json_api:
   address: :7001
-
-dummy:
-  oneshot_common: &oneshot_common
+loglevel: 4
+repos:
+  - name: centos
+    path: /srv/centos
     retry: 5
     retry_interval: 60
```

Comment:
missing of `interval` (seems to be my fault)

Reply:
Just found out that my pyaml format embeds the dummy section into the file. This is not expected. I'll work on this later.